apps/api-extractor/src/analyzer/Span.ts

```diff
@@ -404,39 +431,69 @@ export class Span {
   }
 
   /**
-   * Recursive implementation of `getModifiedText()` and `writeModifiedText()`.
+   * Returns a diagnostic dump of the tree, showing the SpanModification settings for each nodde.
    */
-  private _writeModifiedText(options: IWriteModifiedTextOptions): void {
-    if (this.modification.indentDocComment) {
-      if (options.indentDocCommentState !== IndentDocCommentState.Inactive) {
-        throw new InternalError('indentDocComment cannot be nested');
+  public getModifiedDump(indent: string = ''): string {
```

Comment:
A snapshot test exercising this would be interesting.

Reply:
The test would need to invoke the TypeScript compiler engine on an input string, but otherwise it would be straightforward to set up.  I'm not going to do it here however, since this PR is fixing an somewhat important regression and I want to publish it asap.